tests/integration/long/test_ssl.py

```diff
@@ -42,7 +42,7 @@
 DRIVER_CERTFILE = os.path.abspath("tests/integration/long/ssl/driver.pem")
 DRIVER_CERTFILE_BAD = os.path.abspath("tests/integration/long/ssl/python_driver_bad.pem")
 
-USES_PYOPENSSL = "twisted" in EVENT_LOOP_MANAGER
+USES_PYOPENSSL = "twisted" in EVENT_LOOP_MANAGER or "eventlet" in EVENT_LOOP_MANAGER
 if "twisted" in EVENT_LOOP_MANAGER:
```

Comment:
Do we need to include eventlet here?

Reply:
No actually. Eventlet works fine with the built in `ssl.wrap_socket`, which is used when passing the old-style `ssl_options` arguments, but if constructing a context manually and wrapping it, you have to use pyOpenSSL. Clear as mud, right?

Reply:
I think ;)